app/controllers/hbx_admin_controller.rb

```diff
@@ -8,11 +8,11 @@ class HbxAdminController < ApplicationController
   before_action :set_cache_headers, only: [:edit_aptc_csr]
 
   def registry
-    redirect_to main_app.root_path if ENV['AWS_ENV'] == 'prod'
+    authorize EnrollRegistry, :show?
```

Comment:
@TreyE 
* We are following a pattern of matching the route with the policy method by adding `?` at the end. Can you make the code change to match the same pattern?
* Curious to know why you added two policies instead of one `HbxAdminPolicy`?

Reply:
I can easily make the change.  Typically in ruby the predicate methods end in ? - I was trying to follow that convention.

The reason I made two different policies is because we're accessing two different types of resources - one is testing if someone is allowed to perform administrative APTC actions - and the other is for access to EnrollRegistry settings.  They should, at the end of the day, probably be in different controllers, as they are different resources.  But for now I wanted to set up proper access control on the ability to see the settings - event if eventually it lives somewhere else.

Reply:
Maybe I was not clear on the first point. I am suggesting using the `registry?` method instead of `show?` for the endpoint `registry` for pundit authorization.